### ISSUE TITLE
Logging improvements

### ISSUE THREAD

**Issue by pofider** [jsreport/jsreport#283]
It should be possible to pass basic winston transport settings through jsreport config. 
The main requirement: **allow to simply switch log level to debug in production mode**. 

The first idea how the logger options could look like in complex form

```js
"logger" : { 
  "console": { "transport": "console", "level": "debug" },
  "main": { "transport": "file", "level": "info", "filename": "log.txt" },
  "error": { "transport": "file", "level": "error", "filename": "error.txt" },
}
```

- It should be `jsreport-core` responsibility to apply this config

- The jsreport log should always add default { "transport": "Debug", "level": "debug" }

- user should be able disable the particular log using `"enabled": false`. This includes also the default debug logging

- `jsreport` package should create preset config and **merge it with the one passed to the initialization.** This should lead into the same behavior as right now with ability to customize it through config.

Switching level in console logging in jsreport package could be done like this then. 
```js
"logger" : { 
  "console": { "level": "debug" } 
}
```

Or to add external logging through loggly like this

```js
"logger" : { 
  "loggly": { "module": "winston-loggly", "transport": "Loggly", "auth": "..." } 
}
```
It would require to have package `winston-loggly` but that is obvious.

**Comment by bjrmatos** [comment]
> user should be able disable the particular log using "enabled": false. This includes also the default debug logging

suporting `"silent": true` too just like standard winston's transport options would be nice!

**Comment by pofider** [comment]
Yes, that makes sense. I agree. Thank you

**Comment by bjrmatos** [comment]
available in `1.7.0`
